src/database/schema/UserSchema.ts

```diff
@@ -150,6 +154,10 @@ class UserSchema implements BaseDbSchema {
         this.profilePicture,
         this.bio,
         this.isBanned,
+        this.anon_user_info_emoji_name,
+        this.anon_user_info_emoji_code,
+        this.anon_user_info_color_name,
+        this.anon_user_info_color_code,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of new properties to the array used in the SQL operation should be fine, assuming they are correctly initialized and used elsewhere in the code.
```diff
         this.isBanned,
+        this.anon_user_info_emoji_name,
+        this.anon_user_info_emoji_code,
+        this.anon_user_info_color_name,
+        this.anon_user_info_color_code,
         this.channelId,
         this.userId
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->